js/views/modals/Settings/Advanced/Appearance.js

```diff
@@ -0,0 +1,37 @@
+import app from '../../../../app';
+import loadTemplate from '../../../../utils/loadTemplate';
+import baseVw from '../../../baseVw';
+
+export default class extends baseVw {
+  constructor( options = { } ) {
+    super( options );
+
+    this.options = options;
+    this.settings = app.settings.clone( );
+  }
+
+  getFormData( ) {
+    return super.getFormData( this.$formFields );
+  }
+
+  render( ) {
+    this.settings = app.settings.clone( );
+    const mySettings = this.settings.toModelFormatJSON().appearanceSettings;
+
+    if ( mySettings.windowControlStyle == 'mac' )
+      app.localSettings.set( { 'macStyleWinControls' : true } );
+    else
+      app.localSettings.set( { 'macStyleWinControls' : false } );
```

Comment:
In either case, setting a model should almost never happen in render. It should really happen in save() and can be as simple as passing the results of getFormData() into Model.save().

It's a little more complex if you happen to have a form that's driven by multiple models. In that case, you'd probably want to add an attribute to each form field indicating which model it belongs to. Then you could call getFormData() with different selectors (one for each form) and save each form individually.

It's also a little more complex because in your view's save, you'd need to manage a save call for each model and the fact that one could succeed whereas the other could fail.